ocs_ci/ocs/ui/views.py

```
@@ -1126,6 +1126,70 @@
+    "nodes_selected_summary": (
+        "//p[contains(text(), 'nodes were selected')]"
+        " | //span[contains(text(), 'nodes were selected')]"
+        " | //*[contains(text(), 'nodes were selected')]",
+        By.XPATH,
+    ),
+    # Final "Create storage cluster" submit button at the bottom of the node list
+    "submit_create_storage_cluster_btn": (
+        "//button[normalize-space()='Create storage cluster'"
+        " and not(@data-test='create-storage-cluster')]"
+        " | //button[@data-test='submit-create-storage-cluster']"
+        " | (//button[normalize-space()='Create storage cluster'])[last()]",
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)